src/tespy/connections.py

```diff
@@ -29,6 +29,7 @@
 from tespy.tools.data_containers import dc_prop
 from tespy.tools.data_containers import dc_simple
 from tespy.tools.helpers import TESPyConnectionError
+from tespy.tools.fluid_properties import calc_physical_exergy
```

Comment:
This line needs to be moved one up (between data_containers and helpers) as the imports should be sorted alphabetically.